edge-util/src/Microsoft.Azure.Devices.Edge.Util/Metrics.cs

```diff
@@ -27,6 +31,10 @@ public void Dispose()
 
         public static Option<IMetricsRoot> MetricsCollector { private set; get; }
 
+        private static List<Action> Gauges { set; get; }
+
+        private static object GaugeListLock;
```

Comment:
Super Nit: fields are camelCase (and go before ctor)

Reply:
Fixed.